<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reservationView</class>
 <widget class="QWidget" name="reservationView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="carInfoLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Selected Car</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="startDateEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>108</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
  </widget>
  <widget class="QDateEdit" name="endDateEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>110</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmPushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Confirm Reservation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>450</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="reservationLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Reservation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>End</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
